### PR TITLE
feat: US GPL modal

### PR DESCRIPTION
**Pull request by gillycheesesteak**

### CODE REVIEW

File: src/components/modal/content/DE/parts/Calculator.jsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
 /** @jsx h */
 import { h, Fragment } from 'preact';
 
-import { useCalculator } from '../../../lib';
+import { useCalculator, useContent } from '../../../lib';
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/Calculator.jsx:4]: This triple `../` makes a good case for us using an aliased root import. It's a little hard to follow

File: src/components/modal/content/DE/parts/Calculator.jsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
 const Calculator = () => {
     const { terms, value, isLoading, submit, changeInput } = useCalculator();
+    const { content } = useContent('INST');
 
     return (
         <Fragment>
             <div className="calculator">
-                <h3 className="title">Monatliche Raten berechnen</h3>
+                <h3 className="title">{content.calculator.title}</h3>
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/Calculator.jsx]: Given how often the subfields of `content` and `terms` are called, it would be good to extract them to simplify the code a bit

File: src/components/modal/content/DE/parts/Disclaimer.jsx [hunk 3]
```diff
@@ -1,25 +1,23 @@
 /** @jsx h */
 import { h } from 'preact';
+import { useContent } from '../../../../lib';
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/Disclaimer.jsx:3]: Oof... this quadruple `../` is even worse. We definitely need some root importing

File: src/components/modal/content/DE/parts/Disclaimer.jsx [hunk 4]
```diff
@@ -1,25 +1,23 @@
...
-                : `Anspruchsberechtigte Kunden müssen PayPal ein SEPA-Lastschriftmandat erteilen sowie über ein deutsches PayPal-Privatkonto mit bestätigtem Bankkonto als Zahlungsquelle verfügen. Angaben gemäß § 6a PAngV: effektiver Jahreszins ${offer.apr}%, fester Sollzinssatz ${offer.nominalRate}%, Nettodarlehensbetrag von ${minAmount} EUR bis ${maxAmount} EUR, Laufzeit ${offer.term} Monate. Bonität vorausgesetzt. Kreditgeber: PayPal (Europe) S.à.r.l. et Cie, S.C.A., 22–24 Boulevard Royal, L-2449 Luxembourg. Repräsentatives Beispiel gem. § 6a Abs. 4 PAngV: Nettodarlehensbetrag 1.000,00 EUR, Gesamtbetrag 1.052,54 EUR, monatl. Rate 87,71 EUR, 12 Raten, Laufzeit 12 Monate, fester Sollzinssatz 9,56%, effektiver Jahreszins 9,99%. `;
+        const disclosure = Number(offer.apr.replace(/[,.]/g, '')) === 0 ? content.disclosure0 : content.disclosure;
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/Disclaimer.jsx]: 1. Would be good to abstract `Number(offer.apr.replace(/[,.]/g, '')) === 0` to something like `isZeroApr` just to make this a little clearer. Took me a minute to figure out what this was doing

2. Can `disclosure0` be renamed to `disclosureFor0Apr` or similar? My first thought was this some kind of strangely represented array

File: src/components/modal/content/DE/parts/INST.jsx [hunk 5]
```diff
@@ -3,34 +3,21 @@ import { h } from 'preact';
...
-        <div className="calculator-container">
-            <Calculator country="DE" />
-        </div>
-    </section>
-);
+    return (
+        <section className="content-body">
+            <Carousel items={content.carousel} />
+
+            {/* <!-- Calculator --> */}
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/INST.jsx]: While I usually prefer more comments and not less, this one seems unnecessary. The `Calculator` component and `calculator-container` class make it very obvious

File: src/components/modal/content/DE/parts/TermsTable.jsx [hunk 6]
```diff
@@ -1,33 +1,22 @@
...
-    );
+    const { content } = useContent('INST');
+
+    const genericError = <h3 className="error">{content.terms.genericError}</h3>;
 
     if (terms.error || !terms.maxAmount) {
         return genericError;
     }
 
     if (+terms.amount < terms.minAmount && terms.type === 'pala') {
```
Comment (merlinstardust) [inline src/components/modal/content/DE/parts/TermsTable.jsx]: Similar to above, the subfields of `terms` could be pulled out.

Also, what's the initial `+` do? It's been a while since I have had to use that

File: src/components/modal/content/DE/styles/header.scss [hunk 7]
```diff
@@ -0,0 +1,55 @@
+.header-wrapper {
```
Comment (merlinstardust) [inline src/components/modal/content/DE/styles/header.scss:1]: +1000 for SCSS. So beautiful.

File: src/components/modal/parts/Tabs.jsx [hunk 8]
```diff
@@ -22,14 +23,19 @@ const Tabs = ({ tabs }) => {
         }
     }, [transitionState, initialTab]);
 
+    const hasHeader = tabs.some(tab => Boolean(tab.header));
     // TODO: Accessibility
```
Comment (merlinstardust) [inline src/components/modal/parts/Tabs.jsx:43]: So I know this was already here, but do you know what needs to be done in terms of accessibility and can you add it here with this comment?

I don't think it should be incorporated into your story, but I think if the comment had more detail, the next dev to see it would be more encouraged to try to make this accessible

**Review by merlinstardust — CHANGES_REQUESTED** [review]

File: src/components/modal/content/DE/parts/Calculator.jsx (hunk 1 above)
Comment (Seavenly) [reply src/components/modal/content/DE/parts/Calculator.jsx:4]: We would love to do this, but I believe we are currently limited by the common SDK webpack config that we utilize.
